lib/src/auth/constants/auth_error_code.dart

```diff
@@ -0,0 +1,22 @@
+/// A class containing codes of authentication errors.
+class AuthErrorCode {
+  /// Indicates malformed email address error.
```

Comment:
```suggestion
  /// An error code that indicates a malformed email address error.
```
Please apply to all error codes.